packages/winpanda/extra/src/winpanda/core/cmdconf.py

```diff
@@ -252,76 +253,173 @@ def get_dcos_conf(self):
         """Get the DC/OS aggregated configuration object.
 
         :return: dict, set of DC/OS shared and package specific configuration
-                 objects:
-                     {
-                         'package': {[
-                             {'path': <str>, 'content': <str>},
+                 templates coupled with 'key=value' substitution data
+                 container:
+                 {
+                    'template': {
+                        'package': [
+                            {'path': <str>, 'content': <str>},
                              ...
-                         ]}
-                     }
+                        ]
+                    },
+                    'values': {
+                        key: value,
+                        ...
+                    }
+                 }
         """
 
         dstor_root_url = (
             self.cluster_conf.get('distribution-storage', {}).get(
                 'rooturl', ''
             )
         )
-        dstor_dcoscfg_path = (
+        dstor_linux_pkg_index_path = (
             self.cluster_conf.get('distribution-storage', {}).get(
-                'dcoscfgpath', ''
+                'dcosclusterpkginfopath', ''
             )
         )
+        dcos_conf_pkg_name = 'dcos-config-win'
+        template_fname = 'dcos-config-windows.yaml'
+        values_fname = 'expanded.config.full.json'
+
         # Unblock irrelevant local operations
-        if self.cluster_conf_nop or dstor_dcoscfg_path == 'NOP':
+        if self.cluster_conf_nop or dstor_linux_pkg_index_path == 'NOP':
             LOG.info(f'{self.msg_src}: dcos_conf: NOP')
             return {}
 
-        dcoscfg_url = posixpath.join(dstor_root_url, dstor_dcoscfg_path)
-        dcoscfg_fname = Path(dstor_dcoscfg_path).name
+        # Linux package index direct URL
+        lpi_url = posixpath.join(dstor_root_url, dstor_linux_pkg_index_path)
```

Comment:
You can use `urlparse.urljoin` to join URL's https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urljoin

Reply:
Good point. Thanks. Will change it in some later commit.